### DISCUSSION TITLE
DEV: All about profiles.

### DISCUSSION THREAD

**Discussion by real-yfprojects** [borgbase/vorta#1169] [Ideas]
What can/should profiles be used for? I know the following already:
>Profiles provide a nice way to organize your backups. There are proven use cases, for example some users want to backup some folders in one location and some others in another location.

- Is it intended to use multiple profiles for the same *borg* repository?
- Will *vorta* do the scheduled backups for all profiles or only for the active one?
- And what about multiple repositories for the same profile?

**Comment by real-yfprojects** [comment]
@m3nu

**Reply by real-yfprojects** [reply]
ping @m3nu

**Reply by m3nu** [reply]
Sorry, must have overlooked this. I dont have the final answers. Borgmatic has multiple repos per profile. Maybe a starting point?

**Reply by real-yfprojects** [reply]
> Sorry, must have overlooked this. I dont have the final answers. Borgmatic has multiple repos per profile. Maybe a starting point?

I didn't have a look at borgmatic yet, but I always wondered whether vorta could use it as a backend.
